ex1/ex1.c

```diff
@@ -9,6 +9,21 @@
 int main(void)
 {
     // Your code here
+   int x = 100;
+    printf("hi I am (pid: %d)\n", (int) getpid());
+    int rc = fork();
 
+    if (rc < 0) {
+        fprintf(stderr, "fork failed\n");
+        exit(1);
+    } else if (rc == 0) {
+        printf("me child (pid: %d) and x is %d\n", (int) getpid(), x);
+        x++;
+        printf("child again, x iw now %d\n", x);
+    } else {
+        printf("me parent of %d (pid: %d) and x is: %d\n", rc, (int) getpid(), x);
+        x--;
+        printf("parent again, x is now %d\n", x);   
+    }
     return 0;
```

Comment:
Great job playing around with this and seeing what happens when you change the value of x.